packages/components/src/scroll-lock/test/index.tsx

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { render } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import ScrollLock from '..';
+
+describe( 'scroll-lock', () => {
+	const lockingClassName = 'lockscroll';
+
+	it( 'locks when mounted', () => {
+		expect( document.documentElement ).not.toHaveClass( lockingClassName );
+		render( <ScrollLock /> );
+		expect( document.documentElement ).toHaveClass( lockingClassName );
+	} );
+
+	it( 'unlocks when unmounted', () => {
+		const { unmount } = render( <ScrollLock /> );
+		expect( document.documentElement ).toHaveClass( lockingClassName );
+		unmount();
+		expect( document.documentElement ).not.toHaveClass( lockingClassName );
+	} );
```

Comment:
I wonder if using `toMatchStyleDIffSnapshot` here would make for a better test — basically checking that the difference between locked and unlocked state on `document.body` only produces an `overflow: hidden` style diff.

Otherwise, we could completely change the styles associated with the `lockscroll` CSS class and tests wouldn't break.

Reply:
Not sure how to `toMatchStyleDIffSnapshot` here 🤔 I tried, but failed to get the style. Are you able to take a look at it @ciampo? Either in this PR or as a follow-up.